swarm/network/discovery.go

```diff
@@ -156,19 +156,30 @@ func (msg subPeersMsg) String() string {
 	return fmt.Sprintf("%T: request peers > PO%02d. ", msg, msg.Depth)
 }
 
+// handleSubPeersMsg handles incoming subPeersMsg
+// this message represents the saturation depth of the remote peer
+// saturation depth is the radius within which the peer subscribes to peers
+// the first time this is received we send peer info on all
+// our connected peers that fall within peers saturation depth
+// otherwise this depth is just recorded on the peer, so that
+// subsequent new connections are sent iff they fall within the radius
```

Comment:
iff  means if   and      only        if

Reply:
minor: then I would prefer 'if and only if' because that requires no knowledge of the abbreviation; that has to be written just once and read many times